src/help/changelog.md

```diff
@@ -1,682 +1,29 @@
-#### September 21, 2016
+#### September 30, 2016
 This is the Full Version of the DATA Act Broker and contains everything agencies need to test their data.
```

Comment:
Let's change this to "This version of the DATA Act Broker contains everything agencies need to test the data validation and submission process." to make it consistent with the broker email going out.